tidy3d/web/cli/develop.py

```diff
@@ -633,16 +675,51 @@ def update_notebooks_remote(args=None):
 @click.option(
     "--json-dictionary",
     "-j",
-    type=click.Path(exists=True, file_okay=False, readable=True),
-    default=".",
+    type=click.Path(exists=True, file_okay=True, readable=True),
     help="JSON that contains the docstring version update files.",
 )
-@click.argument("selected_version")
-def replace_in_files_command(directory, json_file_path, selected_version):
+@click.option(
+    "--selected-version",
+    "-v",
+    type=str,
+    help="Version to select from the JSON file",
+)
+@click.option(
+    "--dry-run",
+    type=bool,
+    default=False,
+    help="Dry run the replace in files command.",
+)
+def replace_in_files_command(directory, json_dictionary, selected_version, dry_run):
```

Comment:
might be worth it while we're at it to add type annotations here (and also the return types for the other methods like  `def replace_in_files(...) -> None:`. Thanks!

Reply:
Sorted